src/classes/User.js

```diff
@@ -26,18 +26,30 @@ class User {
     }
   }
 
-  filterByTag(tag) {
-    return this.favoriteRecipes.filter(recipe => {
-      return recipe.tags.includes(tag);
-    });
+  filterByTag(tags) {
+    const newFilterTags = typeof tags === "string" ? [tags] : tags;
+    let filteredRecipes = [];
+    newFilterTags.forEach(tag => {
+      this.favoriteRecipes.forEach(recipe => {
+        if (recipe.tags.includes(tag)) {
+          filteredRecipes.push(recipe)
+        }
+      })
+    })
+    return [...new Set(filteredRecipes)];
   }
 
-  findFavorites(name) {
-    const newSearchText = name.toLowerCase();
+  findFavorites(nameOrIngredient) {
```

Comment:
Ahh I think I can refactor my methods to do this as well .. name or ingredient.  I have them separated right now